samples/bluetooth/central_gatt_write/src/gatt_write_common.c

```diff
@@ -17,6 +17,10 @@
 static uint32_t phy_update_countdown;
 static uint8_t phy_param_idx;
 
+/* Count down number of metrics intervals before performing a param update */
+#define PARAM_UPDATE_COUNTDOWN PHY_UPDATE_COUNTDOWN
```

Comment:
IT'S THE PHYNAL COUNTDOWN
![image](https://github.com/user-attachments/assets/ce952798-7baa-463b-806d-25ab1051a6e2)

(sorry, couldn't help myself)